### DISCUSSION TITLE
Help Needed: Fixing Button Functionality in Next.js with Firebase

### DISCUSSION THREAD

**Discussion by Siddesh0002T** [community/community#148074] [Programming Help]
### Select Topic Area

Question

### Body

Hi Dev Community,

I’m currently building a project using Next.js, TypeScript, Firebase, and Tailwind CSS. I’m encountering issues with the 'Like' and 'Book' buttons.  

Detailed problem and code can be found here: [GitHub Issue #4](https://github.com/RubbySoft/fixmate/issues/4).

Any guidance would be greatly appreciated!

**Comment by mecodeatlas** [comment]
Thanks for posting in the GitHub Community, @Siddesh0002T!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the Programming Help category. I've gone ahead and moved it for you. Good luck!
